Package:     lilo
Subject:     10 fix src makefile
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   no
Last-Update: 2013-06-07

The newer dash shell do not want "-x" inside an makefile.
In the long version it is running without errors. Now
lilo.static will stored to /sbin/ too.
---

diff -urN s09/src/Makefile s10/src/Makefile
--- s09/src/Makefile	2013-06-07 22:46:42.315254388 +0200
+++ s10/src/Makefile	2013-06-07 22:44:13.000000000 +0200
@@ -245,7 +245,10 @@
 
 	strip lilo
 	cp lilo $$DESTDIR$(SBIN_DIR)/lilo
-	[ -x lilo.static ] && strip lilo.static
+	if [ -x lilo.static ]; then \
+	  strip lilo.static; \
+	  cp lilo.static $$DESTDIR$(SBIN_DIR)/lilo.static; \
+	fi
 	cp ../mkrescue $$DESTDIR$(USRSBIN_DIR)/mkrescue
 	cp ../keytab-lilo.pl $$DESTDIR$(USRSBIN_DIR)/keytab-lilo
 
